CHANGELOG

```diff
@@ -1,5 +1,267 @@
 This file contains a summary of changes to the Oppia code base.
 
+v3.0.2 (01 Sep 2020)
```

Comment:
@aks681 Just wanted to confirm that there are 0 new contributors (with at least 1 PR) this month?

Reply:
There were 3 in the release summary that was generated but all 3 were already listed in develop, hence no new additions here.